app/webfront/src/api/index.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+axios.defaults.baseURL = import.meta.env.VITE_API_BASE_URL;
```

Comment:
тут можна задати відносний шлях без сервера (в нас фронт та бек сервери це один сервер)
```suggestion
axios.defaults.baseURL = '/api';
```